README.md

```diff
@@ -274,6 +281,9 @@ Note that `dhcp4_send_hostname` is only supported by the `nm` provider and trans
 to [`ipv4.dhcp-send-hostname`](https://developer.gnome.org/NetworkManager/stable/nm-settings.html#nm-settings.property.ipv4.dhcp-send-hostname)
 property.
 
+Manual DNS configuration can be specified via a list of addresses
```

Comment:
How about:

    The `dns` keyword accepts a list of IP addresses of DNS servers for manual configuration with the list of domains to search after the `dns_search` keyword.

Another possibility would be to add parenthesis to the key words, otherwise it appears to me that the sentence is not correct grammar-wise:

    Manual DNS configuration can be specified via a list of addresses (`dns`) and a list of domains to search (`dns_search`).

Reply:
I would not call it a `keyword`, this would be the first time this term occurs in this document, and would therefore be inconsistent. I think the keys at this level are called `options` or just `keys`.

Reply:
Yes, good catch.

Reply:
I went for

Manual DNS configuration can be specified via a list of addresses
given in the `dns` option and a list of domains to search given in the
`dns_search` option.

But the original version was copied directly from the preexisting text about `address`. So that one is a bit ungrammatical as well.